input/pagecontent/StructureDefinition-au-erequesting-imagingrequest-intro.md

```diff
@@ -0,0 +1,11 @@
+### Usage scenarios
+TBD
+
+### Profile specific implementation guidance
+- The use of coding can vary significantly across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.
+- `ServiceRequest.code`:
+  - **SHOULD** be selected from the preferred RANZCR Radiology Referral value set
+  - if a system is unable to provide a code from the preferred RANZCR Radiology Referral value set , it is preferred to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set over only supplying local codes
+  - coding is preferred over supplying only free text with no codes
+
+<div class="stu-note">A binding for RANZCR Radiology Referral value set is forthcoming.</div>
```

Comment:
Move the STU note to be the top of this file.

Suggest the wording be more:
It is intended that this profile will always support the latest version of the RANZCR Radiology Referral value set in `ServiceRequest.code`. Work is underway to make that available shortly. Implementers are advised to take note that the current binding will be replaced as soon as a version of the  RANZCR Radiology Referral value set is available..

Reply:
Done

Reply:
Noting that formatting `ServiceRequest.code` is not working so removed altogether from the file until I can make it functioning.